src/utilities/testHelpers.tsx

```diff
@@ -1,11 +1,31 @@
 import React, { ReactNode } from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
 import { WrapperComponent } from '@testing-library/react-hooks';
+import faker from 'faker';
+import { ManifestEntry } from '../hooks/useSatelliteManifests';
 
 interface wrapperProps {
   children: ReactNode;
 }
 
+export const createMockManifests = (numManifests: number): ManifestEntry[] => {
```

Comment:
This solutions works, but can we use this https://github.com/thoughtbot/fishery for generating fake data?  Apparently the people that make factory-bot for ruby (the most popular test data generation gem) made this js package recently.  You define factories and can use them for more easily generating individual objects where we only care about a few attributes using `build` or for mass creating lists of objects like this using `buildList`.

Reply:
ok updated that.